ospf6d/ospf6_top.c

```diff
@@ -745,8 +763,10 @@ DEFPY (clear_router_ospf6,
 	ospf6 = ospf6_lookup_by_vrf_name(vrf_name);
 	if (ospf6 == NULL)
```

Comment:
Please add brackets to the `if` part – this is needed according to the coding style guide.

Reply:
done thanks

Reply:
You added the brackets to the wrong place :) (see my comment above)
And now we have a new warning from the CI because of this. Please, add braces here and remove above, and I'll merge.